packages/sdk-v2/src/sdk/analytics/amplitude/api.native.ts

```diff
@@ -1,48 +1,50 @@
-import { Amplitude, Identify } from '@amplitude/react-native'
-import { isFunction, noop } from 'lodash'
+import { Amplitude, Identify } from "@amplitude/react-native";
+import { isFunction, noop } from "lodash";
 
 class Wrapper {
   constructor(instance, identityClass) {
     const initialize = (apiKey, _, options, onReady) => {
-      const { onError = noop } = options || {}
-      const onSuccess = onReady || noop
+      const { onError = noop } = options || {};
+      const onSuccess = onReady || noop;
 
       instance
         .init(apiKey)
         // eslint-disable-next-line require-await
         .then(async initialized => {
           if (!initialized) {
-            throw new Error('Amplitude not initialized !')
+            throw new Error("Amplitude not initialized !");
           }
 
-          onSuccess()
+          onSuccess();
         })
-        .catch(onError)
-    }
+        .catch(onError);
+    };
 
     return new Proxy(this, {
       get: (_, property) => {
-        let propertyValue: any
+        let propertyValue: any;
 
         switch (property) {
-          case 'Identify':
-            return identityClass
-          case 'init':
-            return initialize
-          case 'setVersionName':
-            return noop // there's no setVersionName() on the native SDK
+          case "Identify":
+            return identityClass;
+          case "init":
+            return initialize;
+          case "setVersionName":
+            return noop; // there's no setVersionName() on the native SDK
           default:
-            propertyValue = instance[property]
+            propertyValue = instance[property];
 
             if (isFunction(propertyValue)) {
-              propertyValue = propertyValue.bind(instance)
+              propertyValue = propertyValue.bind(instance);
             }
 
-            return propertyValue
+            return propertyValue;
         }
-      },
-    })
+      }
+    });
   }
 }
 
-export default new Wrapper(Amplitude.getInstance(), Identify)
+const wrapper: Wrapper = new Wrapper(Amplitude.getInstance(), Identify);
+
+export default wrapper;
```

Comment:
revert this. just 

```typescript
new Wrapper(Amplitude.getInstance(), Identify);
```

@L03TJ3 had to solve (or will solve) issues with autogenerating indexes. As i remember we agreed don't use ctix for now. Just exclude ctix calls from package.json scripts:

```json
    "build": "yarn dev:clean && tsc",
    "build:release": "yarn dev:clean && tsc -p tsconfig.release.json",
    "dev:clean": "rm -fr dist types tsconfig.tsbuildinfo",
```

Reply:
reverted, adjusted scritps